vector/src/main/java/im/vector/app/features/lifecycle/VectorActivityLifecycleCallbacks.kt

```diff
@@ -94,9 +95,13 @@ class VectorActivityLifecycleCallbacks constructor(private val popupAlertManager
         val context = activity.applicationContext
         val packageManager: PackageManager = context.packageManager
 
-        // Get all activities from app manifest
+        // Get all activities from element android and android permission controller app
         if (activitiesInfo.isEmpty()) {
             activitiesInfo = packageManager.getPackageInfo(context.packageName, PackageManager.GET_ACTIVITIES).activities
+
+            activitiesInfo += tryOrNull {
+                packageManager.getPackageInfo("com.google.android.permissioncontroller", PackageManager.GET_ACTIVITIES).activities
+            } ?: emptyArray()
```

Comment:
Nit: you could use `.orEmpty()` instead of `?: emptyArray()`

Reply:
> I see [in the doc](https://source.android.com/docs/core/architecture/modular-system/permissioncontroller#package-format) that In Android 11 or higher, the package name is `com.google.android.permission` and not `com.google.android.permissioncontroller`.
> 
> Can you confirm this?

Yes I read this too. My emulator and my pixel 6A are both on Android 13 and the package name is `com.google.android.permissioncontroller`. I suggest adding both (`com.google.android.permission` and  `com.google.android.permissioncontroller`). Are you ok ?